packages/flutter_tools/lib/src/commands/create.dart

```diff
@@ -135,6 +141,16 @@ abstract class Template {
     });
   }
 
+  bool _hasRelativePath(String path1, String path2) {
```

Comment:
I'll add some tests. This method is indented to work with two absolute paths. When split, the paths parts will be `/`, `foo`, ... and `/`, `bar`, ... . I'm trying to verify that the first non-slash part of the paths are the same.